messaging/core.go

```diff
@@ -382,3 +385,33 @@ func (c *Core) decryptMessage(myIdentityKey *ecdsa.PrivateKey, theirPublicKey *e
 
 	return decrypted.DecryptedMessage, nil
 }
+
+func initSDS(logger *zap.Logger) *sds.ReliabilityManager {
+	sds.SetLogger(logger)
+
+	reliabilityManager, err := sds.NewReliabilityManager()
+	if err != nil {
+		logger.Error("SDS: failed to create ReliabilityManager", zap.Error(err))
+		return nil
+	}
+
+	callbacks := sds.EventCallbacks{
+		OnMessageSent: func(messageId sds.MessageID, channelId string) {
+			logger.Debug("SDS: message sent", zap.String("messageId", string(messageId)), zap.String("channelId", channelId))
```

Comment:
You don't need this `SDS: ` prefix in all logs. Instead use logging namespaces
```suggestion
			logger.Debug("message sent", zap.String("messageId", string(messageId)), zap.String("channelId", channelId))
```

Reply:
As I show here: https://github.com/status-im/status-go/pull/7149/files#r2599581106

Reply:
Refactored, but still keep `sds` in the body to help search within log file.